Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteScrollingTreeMac.mm

```diff
@@ -491,20 +491,25 @@ static bool layerEventRegionContainsPoint(CALayer *layer, CGPoint localPoint)
 #endif
 
     if (layersAtPoint.size()) {
-        auto* frontmostLayer = layersAtPoint.first().first;
+        RetainPtr<CALayer> frontmostInteractiveLayer;
         for (size_t i = 0 ; i < layersAtPoint.size() ; i++) {
             auto [layer, point] = layersAtPoint[i];
 
             if (!layerEventRegionContainsPoint(layer, point))
                 continue;
 
+            // At this point, layer is interactive.
```

Comment:
I don't think this comment adds anything; you can read the lines above to see that.